context.go

```diff
@@ -225,11 +260,29 @@ func (context *Context) Close() {
 
 // TotalRuntime returns the cumulated WAF runtime across various run calls within the same WAF context.
 // Returned time is in nanoseconds.
-func (context *Context) TotalRuntime() (overallRuntimeNs, internalRuntimeNs uint64) {
-	return context.totalOverallRuntimeNs.Load(), context.totalRuntimeNs.Load()
+// Deprecated: use Timings instead
+func (context *Context) TotalRuntime() (uint64, uint64) {
+	return uint64(context.timings.RunNs.Load() * int64(time.Nanosecond)), uint64(context.timings.RuntimeNs.Load())
 }
 
 // TotalTimeouts returns the cumulated amount of WAF timeouts across various run calls within the same WAF context.
+// Deprecated: use Timings instead
 func (context *Context) TotalTimeouts() uint64 {
-	return context.timeoutCount.Load()
+	return context.timeBudgetExpirationCount.Load()
+}
+
+// Timings returns the cumulative time spent in various parts of the WAF, all in nanoseconds
```

Comment:
Same as before, the units is a concern of `Timings`, not of this function.

```suggestion
// Timings returns the cumulative time spent in various parts of the WAF
```